app/controllers/service_requests_controller.rb

```diff
@@ -426,6 +426,7 @@ def setup_catalog_calendar
           active_alert.save
           ExceptionNotifier::Notifier.exception_notification(request.env, e).deliver unless request.remote_ip == '128.23.150.107' # this is an ignored IP address, MUSC security causes issues when they pressure test,  this should be extracted/configurable
         end
+      rescue ArgumentError
```

Comment:
`ArgumentError` should probably be handled by the previous rescue block so that we will receive notifications for it.
`rescue Exception, ArgumentError => e`